core/os/input.cpp

```diff
@@ -82,9 +82,20 @@ void Input::_bind_methods() {
 	ObjectTypeDB::bind_method(_MD("action_release","action"),&Input::action_release);
 	ObjectTypeDB::bind_method(_MD("set_custom_mouse_cursor","image:Texture","hotspot"),&Input::set_custom_mouse_cursor,DEFVAL(Vector2()));
 
+	ObjectTypeDB::bind_method(_MD("get_connected_trackers","type"),&Input::get_connected_trackers);
+	ObjectTypeDB::bind_method(_MD("get_tracker_type","device"),&Input::get_tracker_type);
+	ObjectTypeDB::bind_method(_MD("get_tracker_name","device"),&Input::get_tracker_name);
+	ObjectTypeDB::bind_method(_MD("get_tracker_transform","device"),&Input::get_tracker_transform);
+
 	BIND_CONSTANT( MOUSE_MODE_VISIBLE );
 	BIND_CONSTANT( MOUSE_MODE_HIDDEN );
-	BIND_CONSTANT( MOUSE_MODE_CAPTURED );
+	BIND_CONSTANT( MOUSE_MODE_HIDDEN );
```

Comment:
Probably because i haven't rebased with the latest 2.1 yet, I didn't change any of those lines